File: CS-Examples/03_POP3 Client/ConnectToPop3Server.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Email.Smtp;
using Spire.Email.Pop3;

namespace ConnectToPop3Server
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {          
                // Create a POP3 client and connect.
                Pop3Client client = new Pop3Client();
                client.Host = this.HostInput.Text;
                client.Username = this.Username.Text;
                client.Password = this.Password.Text;
                client.Port = int.Parse(this.PortInput.Text);
                client.EnableSsl = true;
                client.Connect();
                MessageBox.Show("Connected Successfully");        
        }
    }
}

File: VB-Examples/03_POP3 Client/ConnectToPop3Server.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Email.Smtp
Imports Spire.Email.Pop3

Namespace ConnectToPop3Server
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			Try
				Dim client As New Pop3Client()
				client.Host = Me.HostInput.Text
				client.Username = Me.Username.Text
				client.Password = Me.Password.Text
				client.Port = Integer.Parse(Me.PortInput.Text)
				client.EnableSsl = True
				client.Connect()
                MessageBox.Show("Connected Successfully")
			Catch ex As Exception
				MessageBox.Show(ex.Message)
			End Try
		End Sub
	End Class
End Namespace